newIDE/app/src/Utils/UsersAutocomplete.js

```diff
@@ -83,7 +83,7 @@ export const UsersAutocomplete = (props: Props) => {
         );
         setUsers(
           Object.keys(userPublicProfilesByIds).map(userId => {
-            const userPublicProfile: UserPublicProfile =
+            const userPublicProfile: UserPublicProfileSearch =
```

Comment:
@AlexandreSi I know this PR is not finished but I still took a look after looking at your backend PR - in the backend the `UserPublicProfileByIds` is indeed returning a `UserPublicProfile` (and not a `UserPublicProfileSearch` - and this is ok!).
So I guess here we need to keep a `UserPublicProfile`. 

Is this overfetching? If it was the search part of the autocomplete yes, but because this is the part used to display usernames and later possibly avatars, I don't think so or that it has an importance.

So I think we can keep UserPublicProfile here. At least does not sounds like a huge overfetching.

Reply:
(Or adapt the backend to return `UserPublicProfileSearch` for the getUserPublicProfileByIds? That's a possibility too, but would be part I guess of an upcoming PR to clean the separation between UserPublicProfile endpoints and UserPublicProfileSearch :))